docs/docs/building-a-web-application.md

```diff
@@ -353,18 +363,18 @@ blog_development=# select * from view_schema.author_records;
 
 We have authors in the database! This means we can also display them in our app:
 
-Change the `app/web.rb`
+Change the `app/web.rb`:
 
 ```ruby
-  get '/authors/id/:aggregate_id' do
+  get '/authors/:aggregate_id' do
```

Comment:
I removed this from the author path to keep it consistent with Rails routes pattern.

Although this isn't Rails, I think it's nice to use something most of Ruby developers are used to it.

Hope it makes sense :)